src/include/libpbs.h

```diff
@@ -338,6 +339,7 @@ struct batch_reply
 #define EXTEND_OPT_NEXT_MSG_PARAM "next_msg_param"
 
 int is_compose(int, int);
+int ps_compose(int, int);
```

Comment:
what is ps_compose to be used for? Peer server compose? Perhaps a comment would help

Reply:
This is mentioned in the function Doxygen.

Reply:
I would like to keep it similar to is_compose(). A short name helps for a function that is used quite frequently.